src/components/Tabbar/Tabbar.tsx

```diff
@@ -23,27 +39,23 @@ export const Tabbar = ({
 }: TabbarProps) => {
   const platform = usePlatform();
 
-  const getItemsLayout = () => {
-    switch (itemsLayout) {
-      case "horizontal":
-      case "vertical":
-        return itemsLayout;
-      default:
-        return React.Children.count(children) > 2 ? "vertical" : "horizontal";
-    }
-  };
+  const _itemsLayout = getItemsLayout(itemsLayout, children);
```

Comment:
Стоит по аналогии с другими местами использовать вот такой нейминг

```suggestion
  const itemsLayout = getItemsLayout(itemsLayoutProp, children);
```